docs/internships/ideas.md

```diff
@@ -39,13 +39,13 @@ Many of you reading this page are interested in participating in our various int
 
 1. Make sure to read the relevant `Introduction` and `Applying` pages first.
    1. GitHub Externship:
-      1. [Introduction](https://docs.talawa.io/docs/internships/github/github-introduction).
-      1. [Application Guide](https://docs.talawa.io/docs/internships/github/github-application-process).
+      1. Introduction
```

Comment:
The github externship section can be completely removed.

Reply:
Will do :)
Also, should I add Outreachy in this list, and link it's introduction and contribution page?

Reply:
@Nitya-Pasrija Yes, please! Good point.